Extension/translations_auto_pr.js

```diff
@@ -13,24 +13,34 @@ const pullRequestTitle = '[Auto] Localization - Translated Strings';
 
 let repoOwner = process.argv[2];
 let repoName = process.argv[3];
-let locProjectName = process.argv[4];
-let authUser = process.argv[5];
-let authToken = process.argv[6];
-let userFullName = process.argv[7];
-let userEmail = process.argv[8];
-let locRepoPath = process.argv[9];
-
-if (!repoOwner || !repoName || !locProjectName || !authUser || !authToken || !userFullName || !userEmail || !locRepoPath) {
-    console.error(`ERROR: Usage: ${path.parse(process.argv[0]).base} ${path.parse(process.argv[1]).base} repo_owner repo_name loc_project_name auth_user auth_token user_full_name user_email loc_repo_path`);
+let authUser = process.argv[4];
+let authToken = process.argv[5];
+let userFullName = process.argv[6];
+let userEmail = process.argv[7];
+let locRootPath = process.argv[8];
+let locSubPath = process.argv[9];
+
+if (!repoOwner || !repoName || !authUser || !authToken || !userFullName || !userEmail || !locRootPath || !locSubPath) {
+    console.error(`ERROR: Usage: ${path.parse(process.argv[0]).base} ${path.parse(process.argv[1]).base} repo_owner repo_name auth_token user_full_name user_email loc_root_path loc_sub_path`);
+    console.error(`   repo_owner - The owner of the repo on GitHub.  i.e. microsoft`);
+    console.error(`   repo_name - The name of the repo on GitHub.  i.e. vscode-cpptools`);
+    console.error(`   auth_user - User account wiith permission to post a pull request against the GitHub repo.`);
+    console.error(`   auth_token - A PAT associated with auth_user.`);
+    console.error(`   user_full_name - A full name to associate with a git commit. (This is replaced by the PR account if commit is squashed.)`);
+    console.error(`   user_email - An email to associate with a git commit. (This is replaced by the PR account if commit is squashed.)`);
+    console.error(`   loc_root_path - The path to the folder with language-specific directories (containing localized xlf files).`);
+    console.error(`   loc_sub_path - A sub-path after the language-specific directory, where the xlf to import is located.  This should not include the name of the xlf file to import.)`);
     return;
 }
 
 console.log(`repoOwner=${repoOwner}`);
 console.log(`repoName=${repoName}`);
-console.log(`locProjectName=${locProjectName}`);
 console.log(`authUser=${authUser}`);
 console.log(`authToken=${authToken}`);
```

Comment:
Credentials/secrets should not appear in logging.

Reply:
They are actually automatically stripped out by the build pipeline. i.e.:

```
authUser=csigs
auth***=***
```

Reply:
Similarly, when passed as an arg to a command line, the echoing of that command line is masked:
```
cd Extension && node ./translations_auto_pr.js microsoft vscode-cpptools csigs *** csigs csigs@users.noreply.github.com "D:\a\1\a/loc" vscode-extensions-localization-export/vscode-extensions && cd ..
```